docs/content/src/learn/guides/services/service-composition/README.md

```diff
@@ -9,9 +9,11 @@ The following are the sections available in this guide.
 - [What you'll build](#what-youll-build)
 - [Prerequisites](#prerequisites)
 - [Implementation](#implementation)
+  - [Creating the project structure](#Creating-the-project-structure)
```

Comment:
Let's remove sub-sections in this TOC. This just needs high level sections. We have it like this in all guides.

Reply:
Corrected.